tools/convert/configData.yaml

```diff
@@ -0,0 +1,1204 @@
+# this contains a list of all of the config items supported by refinery
+# if v1name is missing, it's the same as name
+# if firstversion is missing, it's v1.x (meaning it's part of the legacy config)
+# if lastversion is present, it was valid in that version but not after
+
+# fields
+    # - v1name -- the name of this field in v1 (if empty, same as name)
+    # - v1group -- the group of this field in v1 (if empty, no group)
+    # - name -- the name of this field in v2 (required)
+    # - group -- the group of this field in v2 (if empty, no group)
```

Comment:
When we eventually do a v3 release, how with `v1name`, `v1group`, `name` and `group` interact?  Do we need `v2` prefixes here?

Reply:
It's more my expectation that v1name and v1group would go away after we're not parsing v1 anymore. The point is that name and group are what they say, and the v1* entries are how they used to be. 

In some v3 future, I'd probably add v2name to refer to the past, rather than adding the v2 prefixes to the actual name.